mysql-test/main/analyze_format_json.test

```diff
@@ -267,4 +267,23 @@ where
 set join_cache_level=@tmp;
 drop table t10, t11;
 
+--echo #
+--echo # MDEV-33309: ANALYZE FORMAT=JSON: UPDATE|DELETE don't show r_other_time_ms
+--echo #
+--source include/have_sequence.inc
+create table t1 (pk int primary key, a int);
+insert into t1 select seq, seq from seq_1_to_1000;
+create table t2 like t1;
+insert into t2 select * from t1;
+
+--source include/analyze-format.inc
+analyze format=json 
```

Comment:
Please add `--echo # Top-level query block must have r_table_time_ms and r_other_time_ms`